app/build.gradle

```diff
@@ -35,13 +36,14 @@ dependencies {
   compile 'com.android.support:support-v4:20.+'
   compile 'com.jakewharton:butterknife:5.1.2'
   compile 'com.squareup:otto:1.3.5'
-  compile 'com.squareup.dagger:dagger:1.2.2'
-  provided 'com.squareup.dagger:dagger-compiler:1.2.2'
+  compile 'com.google.dagger:dagger:2.0-SNAPSHOT'
+  compile 'com.google.dagger:dagger-compiler:2.0-SNAPSHOT'
+  compile 'org.glassfish:javax.annotation:10.0-b28'
   compile 'com.squareup.picasso:picasso:2.3.4'
   compile 'com.f2prateek.ln:ln:1.1.1'
   compile 'com.f2prateek.dart:dart:1.1.0'
   compile 'com.github.frankiesardo:android-auto-value:0.1'
-  provided 'com.github.frankiesardo:android-auto-value-processor:0.1'
+  compile 'com.github.frankiesardo:android-auto-value-processor:0.1'
```

Comment:
Thanks! I think I put this in when I didn't realize the glassfish dependency was required (assuming it was a bug with the apt plugin), and forgot to remove it out.